lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -730,6 +818,12 @@ class DecomposeComplexOpsPass
     target.addIllegalOp<AtenArangeStartOp>();
     patterns.add<DecomposeAtenArgMaxOp>(context);
     target.addIllegalOp<AtenArgmaxOp>();
+    patterns.add<DecomposeAtenSquareOp>(context);
+    target.addIllegalOp<AtenSquareOp>();
+    patterns.add<DecomposeAtenVarOp>(context);
+    target.addIllegalOp<AtenVarOp>();
+    patterns.add<DecomposeAtenStdOp>(context);
+    target.addIllegalOp<AtenStdOp>();
```

Comment:
Just driving by and wondering: there aren't any FileCheck tests for the C++ code?

Reply:
yea... we didn't enforce that. Usually just add e2e test. I should add that as a good practice.